.github/workflows/deploy.yaml

```diff
@@ -202,3 +202,45 @@ jobs:
         echo "This build is ready for promotion to prod:" > /tmp/body.txt
         echo "IronCoreLabs/deployments#${{ steps.promote-pr.outputs.pr }}" >> /tmp/body.txt
         gh pr comment -F /tmp/body.txt ${{ steps.software-pr.outputs.pr }}
+
+    # Update the local dev clusters to use the new container.
+    - name: Check out depot
+      uses: actions/checkout@v3
+      with:
+        repository: IronCoreLabs/depot
+        path: depot
+        token: ${{ secrets.WORKFLOW_PAT }}
+    - name: Update local dev clusters
+      id: update-depot
+      working-directory: depot/localk8s
+      run: |
+        set -x
+        for IMG in ${{ inputs.images }} ; do
+          IMG_NAME=$(echo "${IMG}" | sed 's/[^=]*=//')
+          if [ -n "${{ inputs.image-override }}" ] ; then
+            IMG_NAME="${{ inputs.image-override }}"
+          fi
+          if grep "${IMG_NAME}" Tiltfile ; then
+            sed "s,\(FROM ${IMG_NAME}:\)[a-zA-Z0-9_.-]*,\\1${{ inputs.tag }}," < Tiltfile > Tiltfile.new
+            mv Tiltfile.new Tiltfile
+            touch /tmp/changed-depot
+        done
+        if [ -f /tmp/changed-depot ] ; then
+          echo "changed=true" >> "$GITHUB_OUTPUT"
+        fi
```

Comment:
How come instead of `touch /tmp/changed-depot` you don't just do `echo "changed=true" >> "$GITHUB_OUTPUT"` right there and remove the lower `if`? It should be fine if it potentially runs that command multiple times.

Reply:
It just seemed ugly to add that line to `$GITHUB_OUTPUT` multiple times. I agree that it would work the way you describe also. Potayto, potahto?